### PR TITLE
Implement preset_data object and 'sent' field in Polls model

### PR DESCRIPTION
**Pull request by Jakeyrob**
* Update example entries

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -197,12 +197,20 @@ models.sequelize.sync({force: true}).then(function () {
...
   });
 
   models.polls.create({
     type: 'multiChoice',
     lesson_id: 3,
-    answer: 'A'
+    answer: 'A',
+    sent: true,
+    preset_data: JSON.stringify({
+      type: "multiChoice",
```
Comment (iandeboisblanc) [inline server/app.js]: why is type part of preset_data?

Comment (Jakeyrob) [inline server/app.js]: subtype:

Comment (Jakeyrob) [inline server/app.js]: question:

File: server/app.js [hunk 2]
```diff
@@ -197,12 +197,20 @@ models.sequelize.sync({force: true}).then(function () {
...
     type: 'thumbs',
     lesson_id: 2,
+    sent: true
   });
 
   models.polls.create({
     type: 'multiChoice',
     lesson_id: 3,
-    answer: 'A'
+    answer: 'A',
```
Comment (Jakeyrob) [inline server/app.js]: name:

**Pull request merged**